meilisearch/index.py

```diff
@@ -1881,3 +1881,63 @@ def _build_url(
         if primary_key is None and csv_delimiter is None:
             return f"{self.config.paths.index}/{self.uid}/{self.config.paths.document}"
         return f"{self.config.paths.index}/{self.uid}/{self.config.paths.document}?{parse.urlencode(parameters)}"
+
+    # SEARCH CUTOFF MS SETTINGS
+
+    def get_search_cutoff_ms(self) -> int | None:
+        """Get the search cutoff in ms of the index.
+
+        Returns
+        -------
+        search_cutoffms:
```

Comment:
Just to match the other get settings:

```suggestion
        settings:
```

Reply:
done